src/Numerics/Mesh/Topologies.jl

```diff
@@ -17,17 +17,17 @@ export grid1d, SingleExponentialStretching, InteriorStretching
 export basic_topology_info
 
 """
-    AbstractTopology{dim}
+    AbstractTopology{dim, T, nb}
 
-Represents the connectivity of individual elements, with local dimension `dim`.
+Represents the connectivity of individual elements, with local dimension `dim` with `nb` boundary conditions types. The element coordinates are of type `T`.
```

Comment:
My bad, this line should probably be wrapped
```suggestion
Represents the connectivity of individual elements, with local dimension `dim`
with `nb` boundary conditions types. The element coordinates are of type `T`.
```